doc/install.rst

```diff
@@ -83,7 +83,17 @@ Test
 ====
 
 If you are developing for the RAMP package, you will be interested about
-testing your new feature. You can run the test using ``pytest`` from the root
+testing your new feature. First you need to `set up the database <https://paris-saclay-cds.github.io/ramp-docs/stable/setup_server.html#set-up-the-ramp-database>`_ and create the test database::
```

Comment:
this is not the right way to link within sphinx. you need to link to local build.

Reply:
a dev build of the website would point to stable